packages/web/src/pages/profile-page/ProfilePageProvider.tsx

```diff
@@ -288,9 +288,9 @@ class ProfilePage extends PureComponent<ProfilePageProps, ProfilePageState> {
     shouldSetLoading = true
   ) => {
     const params = parseUserRoute(pathname)
-    if (params?.handle) {
+    if (params) {
```

Comment:
just confirming we want this logic change? prob dont want to fetch with null value?